having issue installing it on node-red

farihaPriyanka opened the pull request:
Hi I am trying to install it with node-red latest version  but it is giving error  please help me out .

fauberso committed:
Initial PCA9685 implementation

fauberso committed:
Updated HTML file

fauberso committed:
Corrected naming in HTML file

fauberso committed:
fixed bug in call to registerType

fauberso committed:
Corrected missing comma in json

fauberso committed:
Changed logging

fauberso committed:
Fix: ReferenceError: pwm is not defined

fauberso committed:
Fixed naming if I2C device node

fauberso committed:
Fixed node naming

fauberso committed:
Fixed variable passing

fauberso committed:
Fetch node configuration for values not in msg

fauberso committed:
Modified naming and logging for debugging

fauberso committed:
Corrected way to access server node, added logging

fauberso committed:
corrected debugging

fauberso committed:
Temporary fix until I've figured out node-js logging :)

fauberso committed:
Added more debug logging

fauberso committed:
Added debug logging

fauberso committed:
Additional debug logging

fauberso committed:
Fixed access to pwm property

fauberso committed:
Correct percentage: PCA driver takes values between 0 and 1

fauberso committed:
Cleanup, new Icon

fauberso committed:
Removed extraneous logging, updated documentation. First working
version.

fauberso committed:
Renamed icon

fauberso committed:
Fix PCA9685 name

fauberso committed:
Shortened node name, address is now decimal

fauberso committed:
Improved documentation

fauberso committed:
Added instructions to install i2c first

fauberso committed:
Added physical wiring example

fauberso committed:
Additional physical wiring examples, Added fritzing sources

fauberso committed:
Expanded documentation

fauberso committed:
Added missing picture

aksun1 committed:
update i2c-bus version

aksun1 committed:
fix invalid input type

aksun1 committed:
fix address notation

aksun1 committed:
clarify by improving formatting

aksun1 committed:
fix error in closing method

fauberso committed:
Merge pull request #1 from aksun1/master

Fix i2c-bus version

scrumpydc commented:
latest not working throwing errors through node red
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:196:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Linux 4.19.66-v7+
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/pi/node_modules/node-red-contrib-pca9685/node_modules/i2c-bus
gyp ERR! node -v v10.16.3
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok
npm WARN enoent ENOENT: no such file or directory, open '/home/pi/package.json'
npm WARN pi No description
npm WARN pi No repository field.
npm WARN pi No README data
npm WARN pi No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! i2c-bus@1.1.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the i2c-bus@1.1.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2019-09-22T10_54_10_303Z-debug.log

arrowmeiwaracing committed:
Update package.json

depenciをカット

arrowmeiwaracing committed:
Update package.json

fauberso committed:
Merge pull request #2 from arrowmeiwaracing/master

Bump to later versions of i2c library to support nodejs 10 and 12, thanks @dceejay!

fauberso committed:
Published to NPM as Version 0.1.4

Ultranium committed:
Fix '0' is not being accepted as I2C device number

fauberso committed:
Merge pull request #3 from Ultranium/fix-zero-device-number

Fix '0' is not being accepted as I2C device number

nicomt committed:
Adding power control of channels:
- Allowing to turn channels on and off
- Adding support for "all" channel operations
- Error handling

nicomt committed:
Fix for mistake in on set channel pulse

nicomt committed:
Remember on state for better performance

nicomt committed:
Fix off by one mistake

nicomt committed:
Fixing one more off by one

nicomt committed:
Fix for bug in channel check

nicomt committed:
Delete vscode file pushed by accident

nicomt committed:
Converting spaces to tabs for consistent indentation

nicomt committed:
Changing all to minus one to be compatible with existing validation

nicomt committed:
Remeber channel state across nodes

nicomt committed:
Change power memory faster to prevent race conditions

nicomt committed:
Add power control option to node and docs

nicomt committed:
Making GUI a bit clearer

fauberso committed:
Merge pull request #4 from nicomt/channel-power-control

Adding power control of channels
